src/content/advanced/external-dns/gcp-custom-sa.md

```diff
@@ -0,0 +1,37 @@
+---
+linkTitle: Using a dedicated GCP service account
+title: External DNS using a dedicated GCP service account
+description: This document describes how to configure External DNS to use a dedicated GCP Service Account.
+weight: 25
+menu:
+  main:
+    parent: advanced-external-dns
+last_review_date: 2022-12-21
+user_questions:
+  - How can I override GCP service account used by External DNS?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-cabbage
+---
+
+## Overview
+
+By default, our External DNS App uses the `gcpProject` value to compose the right Service account annotation to correctly authenticate against GCP.
```

Comment:
```suggestion
By default, our External DNS App uses the `gcpProject` value to compose the right service account annotation to correctly authenticate against GCP.
```

I'd propose either `ServiceAccount` or just "service account".

Reply:
In Google Cloud documentation, they refer to it as `service account`. Unless we use it differently, I'd vote we call it the same

Reply:
this refers to the Kubernetes service account, so I'd go for `ServiceAccount`

Reply:
```suggestion
By default, our External DNS App uses the `gcpProject` value to compose the right annotation for the `ServiceAccount` to correctly authenticate against GCP.
```